packages/form-js-playground/src/components/PlaygroundRoot.css

```diff
@@ -79,9 +79,13 @@
   position: relative;
   display: flex;
   height: 100%;
+  width: 250px;
   overflow-y: auto;
   background-color: var(--color-properties-container-background);
-  --properties-panel-width: 200px;
+}
+
+.fjs-pgl-properties-container .fjs-properties-container {
+  --properties-panel-width: 250px;;
```

Comment:
Do we really want to limit the scope? Why can't we define it on `:root`?

Reply:
We are not defining it, we are overriding the one [specified in form-js-editor](https://github.com/bpmn-io/form-js/blob/develop/packages/form-js-editor/assets/form-js-editor.css#L30).

Being specific helps us not to interfere with potential variables from other properties panel libraries. One alternative could be to adjust our CSS variables to be more specific in their names.

Reply:
The reason I've limited the scope is simply that this just did not work previously. The property is used (and defined) for .fjs-properties-container, so it needs to be overridden on a selector of more specificity. Honestly, this kind of CSS variable definition is something that feels very unpredictable and hard to trace. I think in general we probably shouldn't define variables anywhere but at the root, that way the behavior is a bit more predictable.